nikola/nikola.py

```diff
@@ -1711,6 +1711,17 @@ def generic_rss_renderer(self, lang, title, link, description, timeline, output_
             if post.author(lang):
                 rss_obj.rss_attrs["xmlns:dc"] = "http://purl.org/dc/elements/1.1/"
 
+            if itunes:
+                for tag in ('subtitle', 'duration', 'explicit'):
+                    key = 'itunes_{}'.format(tag)
+                    args[key] = post.meta[lang].get(key)
+                args['itunes_author'] = (
+                    post.meta[lang].get('itunes_author')
+                    or post.author(lang))
+                args['itunes_summary'] = post.meta[lang].get('itunes_summary') or data
+                if 'itunes_image' in post.meta[lang]:
+                    args['itunes_image'] = self.url_replacer(post.permalink(), post.meta[lang]['itunes_image'], lang, 'absolute')
```

Comment:
Just keep in mind that the permalink won’t affect parsing of `itunes_image` paths, hey should be relative to site root (in your posts).